docs/testing.md

```diff
@@ -159,7 +159,7 @@ Use `domains-staging.gumroad.com` instead of `domains.gumroad.com` in the DNS re
 
 2. Configure sample-custom-domain.example.com as a custom domain in [gumroad.dev/settings/advanced](https://gumroad.dev/settings/advanced)
 
-3. Remove `Rails.env.development? ||` part from this [line](https://github.com/gumroad/web/blob/3a63b5bd586f38761cc395a3318f34a8216c64b4/lib/gumroad_domain_constraint.rb#L5)
+3. Remove `Rails.env.development? ||` part from this [line](https://github.com/antiwork/gumroad/blob/main/lib/gumroad_domain_constraint.rb#L5)
```

Comment:
The doc said to remove `Rails.env.development? ||` but the line is actually not there. Perhaps the docs need to be updated.

https://github.com/antiwork/gumroad/blob/d0eaae04935d3cc011dfda29efc822e725d7437b/lib/gumroad_domain_constraint.rb#L3-L7